src/accelerate/state.py

```diff
@@ -897,7 +897,10 @@ def __init__(
                 if os.environ.get("ACCELERATE_USE_FSDP", "false") == "true":
                     self.distributed_type = DistributedType.FSDP
                     if self._mixed_precision != "no":
-                        fsdp_plugin.set_mixed_precision(self._mixed_precision)
+                        mixed_precision = self._mixed_precision
+                        if self._mixed_precision == "fp8":
+                            mixed_precision = "bf16"
+                        fsdp_plugin.set_mixed_precision(mixed_precision)
```

Comment:
Why are we setting it to bf16 ?

Reply:
With training in FP8, only certain layers are done in FP8 (such as linear, etc). Otherwise we run bf16 precision for everything else (gradients, backprop, etc).